src/new/wallet.js

```diff
@@ -1562,7 +1660,7 @@ class HathorWallet extends EventEmitter {
    * @param {boolean} [options.signTx] sign transaction instance (default true)
    * @param {boolean} [options.isCreateNFT=false] if the create token is an NFT creation call
    *
-   * @return {CreateTokenTransaction} Promise that resolves with transaction object if succeeds
+   * @return {Promise<CreateTokenTransaction>} Promise that resolves with transaction object if succeeds
```

Comment:
You could use in the docstring above the typedef you created for `CreateTokenOptions`